docs/dev/developer-guides/bridging/l1-l2-interop.md

```diff
@@ -1,72 +1,52 @@
+- The network's censorship resistance.
+- Custom bridges.
+- Multi-layer governing smart contracts.
+- Multi-layer transfers.
 
-## Priority queue
-
-The goal of the priority queue is to provide a censorship-resistant way to interact with zkSync in case the operator becomes malicious or unavailable.
-The way the priority queue works in zkSync Era is very close to how it worked in the previous version of zkSync.
-For the full picture, we first present how the priority queue works on zkSync Lite.
-This gives the rationale for the new design of the priority queue for zkSync Era.
-
-### How it works in zkSync Lite
-
-In the previous version of zkSync, we only had two operations that could be sent to zkSync from L1:
-
-- `Deposit` to bridge funds from Ethereum to zkSync.
-- `FullExit` to bridges the funds back from Ethereum (this is essentially the same as `Withdraw` in zkSync Era).
-
-If users wanted to deposit funds to or withdraw funds from zkSync, they would have to send a transaction request to the smart contract which will then get appended to the queue of priority transactions. The queue has the following rules:
+## L1 to L2 communication
 
-- All transactions are processed sequentially.
-- Each priority operation must be processed by the operator within `X` days since it was submitted to the contract.
+L1 to L2 communication is governed by the [`IZkSync.sol`](https://github.com/matter-labs/v2-testnet-contracts/blob/b8449bf9c819098cc8bfee0549ff5094456be51d/l1/contracts/zksync/interfaces/IZkSync.sol#L4) inherited interfaces.
 
-The first rule is strictly enforced by the smart contract. The second rule may be violated if the operator becomes malicious or unavailable. In case that happens, the system enters 'exodus mode', where no new blocks can be processed and users can withdraw their funds without cooperation from the operator.
+:::tip
+- If you prefer to learn-by-doing, the [cross chain governance tutorial](../../tutorials/cross-chain-tutorial.md) is a practical example of layer interoperability.
+:::
 
-### What changes are needed?
+### Gas estimation
 
-The process described above works well for a system with a small set of relatively light supported operations. zkSync Era supports general smart contract computation, and thus some principles had to be changed in order to preserve the stability of the network.
+The system processes gas estimation for transactions implicitly. However, it is also possible to implement the gas estimation processes explicitly.
 
-Firstly, all transactions need to be supported by the priority queue. Users may have their funds locked on an L2 smart contract, and not on their own L2 account. Therefore before moving their funds to L1, they need to send an `Execute` transaction to the zkSync network to release the funds from that smart contract first.
+:::tip L1 to L2 gas estimation for transactions
+- Gas is measured in amount and not ERC20 token value so actual costs can vary.
```

Comment:
> Gas is measured in amount and not ERC20 token value so actual costs can vary

I am not sure what this means

Reply:
amount == number of gas

Reply:
To remind you that I also had an issue with this phrasing. Maybe better to modify

Reply:
try now